docs/reference/searchable-snapshots/apis/clear-cache.asciidoc

```diff
@@ -0,0 +1,76 @@
+[role="xpack"]
+[testenv="enterprise"]
+[[searchable-snapshots-api-clear-cache]]
+=== Clear cache API
+++++
+<titleabbrev>Clear cache</titleabbrev>
+++++
+
+experimental[]
+
+Clear the cache of searchable snapshots.
```

Comment:
If this is the same as the shared cache, I'd include an xref.
```suggestion
Clears indices and data streams from the <<shared-cache,shared searchable
snapshot cache>>.
```

Reply:
It will be once #71933 is merged